wolfssl/wolfcrypt/types.h

```diff
@@ -898,7 +902,11 @@ typedef struct w64wrapper {
         #endif
         #if defined(OPENSSL_ALL) || defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)
         #define XISALNUM(c)     isalnum((c))
-        #define XISASCII(c)     isascii((c))
+        #if defined(__ORBIS__) || defined(__PROSPERO__) /* PlayStation 4 and 5 */
```

Comment:
Consider also adding another new macro `|| defined(USE_WOLF_ISASCII)` or ` || defined(NO_STDLIB_ISASCII)`.

Reply:
Can you add one of these macros `|| defined(NO_STDLIB_ISASCII)` as well to allow someone the option to force the macro based `XISASCII`?

Reply:
Done, also removed PS-related highly confidential "codenames" since we had some issues with getting approval for it.